docs/hugo/content/introduction/authentication.md

```diff
@@ -14,7 +14,8 @@ Azure Service Operator supports three different styles of authentication today.
 ### Prerequisites
 1. An existing Azure Service Principal.
 
-To use Service Principal authentication, specify an `aso-controller-settings` secret with the `AZURE_CLIENT_ID` and `AZURE_CLIENT_SECRET` keys set.
+To use Service Principal authentication, specify an `aso-controller-settings` secret with the `AZURE_CLIENT_ID` and `AZURE_CLIENT_SECRET` keys set. This Service Principal is used as a default global credential and is optional. However, it is only suggested to exclude above in [single-operator-multitenancy](https://azure.github.io/azure-service-operator/introduction/multitenant-deployment/#single-operator-multitenancy-default) mode
```

Comment:
Consider rephrasing the second sentence here to: "This Service Principal is used as a default global credential and is optional. If you do not specify a global credential, you must follow the instructions in [single-operator-multitenancy](https://azure.github.io/azure-service-operator/introduction/multitenant-deployment/#single-operator-multitenancy-default) to specify per-namespace or per-resource credentials"